src/lib/Tests/Permission/PermissionCheckerTest.php

```diff
@@ -0,0 +1,184 @@
+                            new Policy(['limitations' => [new Limitation\LanguageLimitation(['limitationValues' => [2, 3]])]]),
+                        ],
+                    ],
+                ],
+                'anyClassName',
+                [],
+            ],
+            'onePolicyOneLimitation' => [
+                [
+                    [
+                        'limitation' => null,
+                        'policies' => [
+                            new Policy(['limitations' => [new Limitation\ContentTypeLimitation(['limitationValues' => [2, 3]])]]),
+                        ],
+                    ],
+                ],
+                Limitation\ContentTypeLimitation::class,
+                [2, 3],
+            ],
+            'twoPoliciesWihOneLimitation' => [
+                [
+                    [
+                        'limitation' => null,
+                        'policies' => [
+                            new Policy(['limitations' => [new Limitation\ContentTypeLimitation(['limitationValues' => [2, 3]])]]),
+                            new Policy(['limitations' => [new Limitation\LanguageLimitation(['limitationValues' => [2, 3]])]]),
+                        ],
+                    ],
+                ],
+                Limitation\ContentTypeLimitation::class,
+                [2, 3],
+            ],
+            'twoPoliciesWihOneLimitationAndRoleLimitation' => [
+                [
+                    [
+                        'limitation' => new Limitation\SectionLimitation(['limitationValues' => [44]]),
+                        'policies' => [
+                            new Policy(['limitations' => [new Limitation\ContentTypeLimitation(['limitationValues' => [2, 3]])]]),
+                            new Policy(['limitations' => [new Limitation\LanguageLimitation(['limitationValues' => [2, 3]])]]),
+                        ],
+                    ],
+                ],
+                Limitation\SectionLimitation::class,
+                [44],
+            ],
+            'noPoliciesAndRoleLimitation' => [
+                [
+                    [
+                        'limitation' => new Limitation\SectionLimitation(['limitationValues' => [44]]),
+                        'policies' => [],
+                    ],
+                ],
+                Limitation\SectionLimitation::class,
+                [],
+            ],
+            'policyWithoutLimitationAndWithRoleLimitation' => [
+                [
+                    [
+                        'limitation' => new Limitation\SectionLimitation(['limitationValues' => [44]]),
+                        'policies' => [new Policy()],
+                    ],
+                ],
+                Limitation\SectionLimitation::class,
+                [44],
+            ],
+        ];
+    }
+
+    /**
+     * @param int $id
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
+     */
+    private function generateUser(int $id = null): User
```

Comment:
whats the point of default `null` value?

Reply:
you are right, it is useless here, fixed